src/components/Repos/Repo.js

```diff
@@ -1,45 +1,72 @@
 import { useState, useEffect } from 'react'
-import { useParams } from "react-router-dom"
+import { useParams, Link } from "react-router-dom"
 
 function Repo () {
-  const [repo, setRepo] = useState({})
+  const [repo, setRepo] = useState({notes: []})
   const [notFound, setNotFound] = useState(false)
 
-  const params = useParams()
-  console.log(params)
+  const {username, reponame} = useParams()
 
   useEffect(() => {
-    //  https://api.github.com/repos/OWNER/REPO
-    fetch(`https://api.github.com/repos/${params.username}/${params.reponame}`)
-      .then(res => res.json()) // read the response format which is stored in JSON
-      .then(data => {
-      if (data.message === 'Not Found') {
+    const getRepoAndNotes = async () => {
```

Comment:
I noticed that you used async-await instead of then() method. Is there any difference between them and any reason why one is preferred over the other?